tests/unit_tests/include/test_helper.hpp

```diff
@@ -55,61 +55,87 @@
 
 #if defined(ENABLE_MKLCPU_BACKEND) || defined(ENABLE_NETLIB_BACKEND)
 #ifdef ENABLE_MKLCPU_BACKEND
+#define TEST_RUN_INTELCPU_SELECT_NO_ARGS(q, func) \
+    func(oneapi::mkl::backend_selector<oneapi::mkl::backend::mklcpu>{ q })
 #define TEST_RUN_INTELCPU_SELECT(q, func, ...) \
     func(oneapi::mkl::backend_selector<oneapi::mkl::backend::mklcpu>{ q }, __VA_ARGS__)
 #else
+#define TEST_RUN_INTELCPU_SELECT_NO_ARGS(q, func) \
+    func(oneapi::mkl::backend_selector<oneapi::mkl::backend::netlib>{ q })
 #define TEST_RUN_INTELCPU_SELECT(q, func, ...) \
     func(oneapi::mkl::backend_selector<oneapi::mkl::backend::netlib>{ q }, __VA_ARGS__)
 #endif
 #else
+#define TEST_RUN_INTELCPU_SELECT_NO_ARGS(q, func)
 #define TEST_RUN_INTELCPU_SELECT(q, func, ...)
 #endif
 
 #ifdef ENABLE_MKLGPU_BACKEND
+#define TEST_RUN_INTELGPU_SELECT_NO_ARGS(q, func) \
+    func(oneapi::mkl::backend_selector<oneapi::mkl::backend::mklgpu>{ q })
 #define TEST_RUN_INTELGPU_SELECT(q, func, ...) \
     func(oneapi::mkl::backend_selector<oneapi::mkl::backend::mklgpu>{ q }, __VA_ARGS__)
 #else
+#define TEST_RUN_INTELGPU_SELECT_NO_ARGS(q, func)
 #define TEST_RUN_INTELGPU_SELECT(q, func, ...)
 #endif
 
 #ifdef ENABLE_CUBLAS_BACKEND
+#define TEST_RUN_NVIDIAGPU_CUBLAS_SELECT_NO_ARGS(q, func) \
```

Comment:
If this macro is not used by cuBLAS backend why it's added for DFT PR?
And I have the same question to other BLAS, LAPACK, RNG specific backends.

Reply:
I've removed the macro for anything other than MKLGPU and MKLCPU in [f7566e1](https://github.com/oneapi-src/oneMKL/pull/261/commits/f7566e176c1cb7b30c5b8757acb43c2483fe3967).